libseccomp-sys/src/lib.rs

```diff
@@ -223,6 +225,10 @@ pub const SCMP_ARCH_S390X: u32 = 0x80000016;
 pub const SCMP_ARCH_PARISC: u32 = 0xf;
 pub const SCMP_ARCH_PARISC64: u32 = 0x8000000f;
 pub const SCMP_ARCH_RISCV64: u32 = 0xc00000f3;
+pub const SCMP_ARCH_LOONGARCH64: u32 = 0xc0000102;
+pub const SCMP_ARCH_M68K: u32 = 0x4;
+pub const SCMP_ARCH_SHEB: u32 = 0x2a;
+pub const SCMP_ARCH_SH: u32 = 0x4000002a;
```

Comment:
Maybe good to keep them in the same order as in seccomp.h

Reply:
fixed